### PR TITLE
add storage vm to ost runs

### PR DESCRIPTION
**Pull request by hbraha**
Currently, Except for HE, all suites use engine VM for storage,
add a separate VM for storage.

### CODE REVIEW

File: common/deploy-scripts/setup_storage.sh [hunk 1]
```diff
@@ -52,8 +52,9 @@ set_selinux_on_nfs() {
 }
 
 setup_iscsi() {
-    # this is ugly, assumes that dedicated storage VMs (ost-[suite]-storage) use their primary network as storage network, and VMs with co-located engine have a dedicated storage network on eth1 (like basic-suite-master).
-    if [[ $(hostname) == *"-storage"* ]]; then
+    # this is ugly, assumes that dedicated storage VMs (ost-[suite]-storage) use their primary network as storage network only on HE suite, and VMs with co-located engine have a dedicated storage network on eth1 (like basic-suite-master).
+    echo $SUITE
+    if [[ $(hostname) == *"-storage"* ]] && [[ $(hostname) == *"he-"*  ]]; then
```
Comment (tinez) [inline common/deploy-scripts/setup_storage.sh:57]: Maybe it would be better to add the management interface to storage VM in HE suite too?

Comment (michalskrivanek) [reply common/deploy-scripts/setup_storage.sh:57]: it would be definitely less confusing

File: common/init-configs/1_engine_2_hosts.json [hunk 2]
```diff
@@ -54,6 +56,30 @@
       }
     }
   },
+  "storage": {
```
Comment (michalskrivanek) [inline common/init-configs/1_engine_2_hosts.json:59]: you should drop these from the engine VM then

**Review by michalskrivanek — CHANGES_REQUESTED** [review]

File: common/init-configs/1_engine_2_hosts.json (hunk 2 above)
Comment (tinez) [reply common/init-configs/1_engine_2_hosts.json:59]: If we do this in this PR, we'll break OST. Only once all the suites are migrated to use the new storage VM we can do that

File: common/deploy-scripts/setup_storage.sh (hunk 1 above)
Comment (tinez) [reply common/deploy-scripts/setup_storage.sh:57]: Let's handle this in subsequent patches

**Review by tinez — APPROVED** [review]

**Review by michalskrivanek — APPROVED** [review]

**Pull request merged**